### PR TITLE
Supply web driver instance when saving png files

### LINKED ISSUE #1357: Dark monitor crashing due to reported missing env packages
_Issue [JWQL-138](https://jira.stsci.edu/browse/JWQL-138) was created on JIRA by [Bryan Hilbert](https://jira.stsci.edu/secure/ViewProfile.jspa?name=hilbert):_

On Sept 13, the dark monitor crashed while trying to create png files. The error statement reports:

"RuntimeError: Neither firefox and geckodriver nor a variant of chromium browser and chromedriver are available on system PATH. You can install the former with 'conda install -c conda-forge firefox geckodriver'."

However, I've confirmed that both firefox and geckodriver are present in the jwql-3.9 environment. The environment was just deleted and re-created from scratch last week. Could it be that the web service is working with an old version of the environment? Except that both packages were present in the old version of the environment as well.

### PR DESCRIPTION
**Pull request by bhilbert4**
Resolves #1357 

With selenium > 4.9.1 it looks like we need to supply a web driver instance to bokeh's export_png() when saving png files. This PR creates a small wrapper function around Bokeh's export_png() that creates the web driver and passes it to export_png.

### CODE REVIEW

**Comment by bhilbert4** [comment]
Successful testing here as well. @mfixstsci this is ready for review.

File: jwql/tests/test_utils.py [hunk 1]
```diff
@@ -479,6 +483,17 @@ def test_filesystem_path():
     assert check == location
 
 
+def test_save_png():
```
Comment (mfixstsci) [inline jwql/tests/test_utils.py:486]: Thanks for the test!

File: jwql/utils/utils.py [hunk 2]
```diff
@@ -749,13 +749,34 @@ def read_png(filename):
...
         # with a lower-left origin
-        view[:,:,:] = np.flipud(np.asarray(rgba_img))
+        view[:, :, :] = np.flipud(np.asarray(rgba_img))
     else:
         view = None
     # Return the 2D version
     return img
 
 
+def save_png(fig, filename=''):
```
Comment (mfixstsci) [inline jwql/utils/utils.py:759]: Thanks, this looks great. Is this just patching function until we can get the bokeh version updated? Or are we looking at having to keep this around long term?

**Review by mfixstsci — APPROVED** [review]

File: jwql/utils/utils.py (hunk 2 above)
Comment (bhilbert4) [reply jwql/utils/utils.py:759]: I haven't checked Bokeh > 3.0 to see, but my guess is that this will be a long term fix.

**Comment by mfixstsci** [comment]
Thanks @bhilbert4

**Pull request merged**